cmake/podioBuild.cmake

```diff
@@ -50,7 +50,7 @@ macro(podio_set_compiler_flags)
   # build Catch2 locally without them
   SET(CXX_FLAGS_CMAKE_DEFAULTS "${CMAKE_CXX_FLAGS}")
 
-  SET(COMPILER_FLAGS -Wshadow -Wformat-security -Wno-long-long -Wdeprecated -fdiagnostics-color=auto -Wall -Wextra -pedantic -Weffc++)
+  SET(COMPILER_FLAGS -Wshadow -Wformat-security -Wno-long-long -Wdeprecated -fdiagnostics-color=auto -Wall -Wextra -pedantic -Wno-error=deprecated-declarations -Wno-error=shadow -Wno-error=deprecated-copy-dtor)
```

Comment:
Are these mainly for decluttering the compiler output during development, or are some of these necessary for rootcling?

Reply:
That was necessary for rootcling debugging. Not final yet